svgs/intesa_sanpaolo.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none" stroke="#000"><path stroke-width="12" d="M30 40a10 10 0 0 1 10-10h112c5.52 0 10 4.48 10 10v112c0 5.52-4.48 10-10 10H40c-5.52 0-10-4.48-10-10z"/><path stroke-linecap="round" stroke-linejoin="round" stroke-width="8" d="M70.68 123.37V97.23c-.54-7.11-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13m63.47.01V97.23c-.54-7.12-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13m63.47.01V97.23c-.54-7.12-6.08-12.6-12.74-12.6s-12.2 5.48-12.74 12.6v26.13M146.71 72H45.29"/></svg>
\ No newline at end of file
+<svg width="192" height="192" viewBox="0 0 192 192" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M22 58h148M22 98v36m58-36v36m58-36v36M54 98v36m58-36v36m58-36v36M54 98a16 16 0 1 0-32 0m90 0a16.002 16.002 0 0 0-16-16 16 16 0 0 0-16 16m90 0a16.002 16.002 0 0 0-16-16 16.002 16.002 0 0 0-16 16" stroke="#000" stroke-width="12" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
The original icon features the box around the main part. So I think it should be kept.